src/app/features/event/event-card/event-card.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="event-card">
+  <div class="event-card__info">
+    <img class="info__event-img" src="assets/eventCard.png" alt="event image" />
+    <div class="info__event-details">
+      <h3 class="font-heading-3">{{ eventCard.eventInfo.name }}</h3>
```

Comment:
Tak się zastanawaim nad tym `h3`
jak będzie już widok wydarzenia to będzie pierwszy nagłówek na stronie, więc czy nie lepiej byłoby dać `h1`?

Reply:
Element h1 z klasą heading-3? Wydaję mi się, że powinniśmy trzymać się naszego designu i niech klasy odpowiadają elementom.